apps/website/src/utils/locale.ts

```diff
@@ -0,0 +1,33 @@
+// TODO: straight copy of the same file in the UI package
```

Comment:
Why the file from the UI package isn't used?

Reply:
I did not want to crate an outlier-dependency between ui and website that is only needed for Gatsby.